config/default.yml

```diff
@@ -174,6 +174,18 @@ Bundler/GemComment:
   IgnoredGems: []
   OnlyFor: []
 
+Bundler/GemVersionDeclaration:
+  Description: 'Requires or prohibits gem version declarations.'
+  Enabled: false
+  VersionAdded: '<<next>>'
+  EnforcedStyle: 'required'
+  SupportedStyles:
+    - 'required'
+    - 'prohibited'
```

Comment:
I think we've typically used "forbidden" in the codebase, but you'll have to check this.

Reply:
Forbidden definitely seems to be more common than prohibited, good to know!